OWNERS

```diff
@@ -2,7 +2,7 @@
 // https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
 
 {
-  reviewerTeam: 'ampproject/reviewers-amphtml',
+  reviewerPool: ['ampproject/reviewers-amphtml', 'renovate-bot'],
```

Comment:
```suggestion
  reviewerPool: ['ampproject/reviewers-amphtml', 'renovate-approve'],
```

Reply:
If I'm understanding correctly, `renovate-bot` is the PR author. The PRs are approved by `renovate-approve`.

Reply:
The GH account associated with the app is `@renovate-bot` while the name of the bot that's doing the approving is `renovate-approve`. If the approver identified by owners bot is what appears on GH, then your suggested change is the right one to make.